.theme/pages/demo/pinia.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div>Titles: {{ store.titles }}</div>
+</template>
+<script setup>
+import { usePiniaStore } from "~/store/piniaStore";
+
+const store = usePiniaStore();
+const titles = ["title1", "title2", "title3"];
+
+store.setTitles(titles);
```

Comment:
this sets titles each time it loads, making the demo meaningless. let's check if titles is null, if so set them